internal/configs/oidc/oidc.conf

```diff
@@ -8,7 +8,7 @@
 
     location = /_jwks_uri {
         internal;
-        proxy_cache jwk;                              # Cache the JWK Set received from IdP
+        proxy_cache jwk;                              # Cache the JWK Set recieved from IdP
```

Comment:
This line is just bring in a misspelling - it should be removed.

Reply:
lol I just glanced over it and I thought it was correcting it 😅